### PR TITLE
added Malaysia [ms_MY] locale

### PR DESCRIPTION
**Pull request by kenfai**
Added Malaysia [ms_MY] locale provider for Address, Company, Miscellaneous, Payment, Person, and PhoneNumber.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1284,6 +1284,93 @@ echo $faker->bankAccountNumber // "LT300848876740317118"
...
+echo $faker->streetAddress; // "No. 123, Jalan Utama 7"
+
+// Generates a random Malaysian township
+echo $faker->township; // "Taman Bahagia"
+
+// Generates a random Malaysian town address with matching postcode and state
+echo $faker->townState; // "55100 Bukit Bintang, Kuala Lumpur"
+
+// Generates a random Malaysian address
+echo $faker->address; // "No. 123, Jalan Utama 7, Taman Bahagia, 55100 Bukit Bintang, Kuala Lumpur"
```
Comment (fzaninotto) [inline readme.md]: please don't document formatters that are not specific to your locale

File: src/Faker/Provider/ms_MY/Address.php [hunk 2]
```diff
@@ -0,0 +1,1340 @@
...
+        '{{streetPrefix}} {{streetSuffix}} %',
+        '{{streetPrefix}} {{streetSuffix}} %/%',
+        '{{streetPrefix}} {{streetSuffix}} %/%#',
+        '{{streetPrefix}} {{streetSuffix}} %/%?',
+        '{{streetPrefix}} {{streetSuffix}} %/%#?',
+        '{{streetPrefix}} {{streetSuffix}} %?',
+        '{{streetPrefix}} {{streetSuffix}} %#?',
+    );
+
+    protected static $townshipFormats = array(
```
Comment (fzaninotto) [inline src/Faker/Provider/ms_MY/Address.php]: there is way too much data on this provideR. Faker doesn't look to be exhaustive, or to copy the geographic census of every country. Please provide at most 300 township names, city names, and street names.

File: src/Faker/Provider/ms_MY/Company.php [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    {
+        return static::randomElement(static::$companies);
+    }
+
+    /**
+     * Return a random industry
+     * 
+     * @example 'Automobil'
+     */
+    public static function industry()
```
Comment (fzaninotto) [inline src/Faker/Provider/ms_MY/Company.php]: what's an industry?

Comment (kenfai) [reply src/Faker/Provider/ms_MY/Company.php]: Category or area of business. Is any changes necessary here?

File: src/Faker/Provider/ms_MY/Address.php (hunk 2 above)
Comment (kenfai) [reply src/Faker/Provider/ms_MY/Address.php]: township names, city names, and street names data greatly reduced in latest commit.

File: readme.md (hunk 1 above)
Comment (kenfai) [reply readme.md]: ok they have been removed as requested.

File: src/Faker/Provider/ms_MY/Address.php (hunk 2 above)
Comment (fzaninotto) [reply src/Faker/Provider/ms_MY/Address.php]: Still too much granularity. I suggest the class only exposes one `city` formatter, no need to expose `townJohor` etc as public methods. And that means you can use an associative array to define the data for cities and townships and postcodes in a more concise way.

Comment (kenfai) [reply src/Faker/Provider/ms_MY/Address.php]: Suggestion taken. The data are now in associative arrays with single public method for each.

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**